### PR TITLE
Update emergent architecture closes #71

### LINKED ISSUE #71: Update the Emergent Architecture Design Report.

### PR DESCRIPTION
**Pull request by Jaapp-**
Updated the emergent architecture design.

### CODE REVIEW

**Comment by jwgmeligmeyling** [comment]
Looks good! :+1:

File: Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex [hunk 1]
```diff
@@ -153,7 +157,8 @@ \section{Persistent data management}
 
 \par 
 Each room has an unique room identifier, which is used to find the room requested from the API call.
-We also store basic properties: The name of the room to display on the frontend, the currently playing song and the duration of the song currently playing.
+We also store some properties: The name of the room to display on the frontend, the currently playing song including its metadata, 
+and a playing queue and history of played songs.
```
Comment (eanker) [inline Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex:162]: Replace , and" with just and. I have been thaught ", and" is wrong.

**Comment by eanker** [comment]
You might want to update the erDiagram, because we changed the db.

File: Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex (hunk 1 above)
Comment (Jaapp-) [inline Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex:162]: It's OK in American English.
https://en.wikipedia.org/wiki/Serial_comma

Comment (jwgmeligmeyling) [inline Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex:162]: In British as well ;) the Oxford comma

File: Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex [hunk 2]
```diff
@@ -90,13 +90,17 @@ \subsection{Moodcat Backend}
...
 \item the \textit{REST API} which is used for communication about rooms and songs between the backend and frontend;
-\item the \textit{music matching algorithm} which is used to generate music suggestions for rooms and to find a room for a user.
+\item the \textit{Music Matching Algorithm} which is used to generate music suggestions for rooms and to find a room for a user.
 \end{itemize}
 In the following paragraphs we will explain the last two subcomponents.
 
 \subsubsection{REST API}
 The backend keeps track of which rooms are active and what song is currently played in the room.
-The API can process user votes and classifications in order to improve the system.
+When the backed is started, the backend retrieves rooms from the database and creates an instances for each of those.
```
Comment (TimvdLippe) [inline Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex]: ",the backend creates a room instance for each room existing in the database."

File: Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex [hunk 3]
```diff
@@ -90,13 +90,17 @@ \subsection{Moodcat Backend}
...
+The backend keeps track of a current song, a history of previously played songs and a playing queue for future tracks to play.
```
Comment (TimvdLippe) [inline Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex]: "For each room instance, the backend keeps track of the current song, the history" (replace a with the and state that the room instances store them)

**Comment by eanker** [comment]
LGTM

**Pull request merged**